packages/video_player/video_player/pubspec.yaml

```diff
@@ -32,6 +34,8 @@ dev_dependencies:
   flutter_test:
     sdk: flutter
   pedantic: ^1.8.0
+  pigeon:
+    path: ../../../../packages/packages/pigeon
```

Comment:
We can probably just reference a version here right? If we were to actually check it in.

Reply:
I'll change that when i switch from being a draft.